console/src/components/CompactMusic/CompactMusic.js

```diff
@@ -42,17 +41,21 @@ const CompactMusicBase = kind({
 	},
 
 	render: (props) => {
+		const audioFiles = [
+			'https://sampleswap.org/mp3/artist/254731/BossPlayer_Your-Right-Here-160.mp3',
+			'https://sampleswap.org/mp3/artist/78152/HiatusManJBanner_Show-Stopper-160.mp3',
+			'https://sampleswap.org/mp3/artist/47067/DJ-Masque_Oceanic-Dawn-160.mp3',
+			'https://sampleswap.org/mp3/artist/26546/benzoul_lovevoodoo-160.mp3',
+			'https://sampleswap.org/mp3/artist/19139/MarkNine_In-my-Place-160.mp3',
+			'https://sampleswap.org/mp3/artist/47067/DJ-Masque_Dont-Forget-To-Be-Yourself-160.mp3'
+		];
+
+		const source = audioFiles.map((audioFile, index) => (<source key={index} src={audioFile} type="audio/mp3" />));
+
 		return (
 			<Widget {...props} icon="music" title="Listen" description="Listen to your favorite tunes" view="radio" align="center center">
```

Comment:
Please order this props.
Also, I think we can remove the commented `PlaybackControls` and `Button` import if they are not used anymore